### PR TITLE
Add TextService:GetTextSize()

### PR DESCRIPTION
**Pull request by aportner**

### CODE REVIEW

File: lib/instances/TextService.lua [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+function TextService.prototype:GetTextSize(text, fontSize, font, frameSize)
+    if typeof(text) ~= "string" then
+        error("text must a string", 2)
+    elseif typeof(fontSize) ~= "number" then
+        error("fontSize must a number", 2)
+    elseif typeof(font) ~= "number" then
+        error("font must an Enum.Font", 2)
+    elseif typeof(frameSize) ~= "Vector2" then
+        error("frameSize must be a Vector2", 2)
+    end
```
Comment (LPGhatguy) [inline lib/instances/TextService.lua]: Indented with spaces!

File: lib/instances/TextService.lua [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+
+    local width = string.len(text)
```
Comment (LPGhatguy) [inline lib/instances/TextService.lua]: It might be more idiomatic to just use `text:len()` here.

File: lib/instances/TextService.lua [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
 local TextService = BaseInstance:extend("TextService")
 
+-- right now this is just a lazy [fontSize * len(text), fontSize] clipped by frameSize
+function TextService.prototype:GetTextSize(text, fontSize, font, frameSize)
+	if typeof(text) ~= "string" then
+		error("text must a string", 2)
+	elseif typeof(fontSize) ~= "number" then
+		error("fontSize must a number", 2)
+	elseif typeof(font) ~= "number" then
+		error("font must an Enum.Font", 2)
```
Comment (LPGhatguy) [inline lib/instances/TextService.lua]: You accidentally a word here.

Comment (aportner) [reply lib/instances/TextService.lua]: lol

**Pull request merged**